locations/spiders/averitt.py

```diff
@@ -1,69 +1,118 @@
-import re
-
 import scrapy
-
 from locations.items import GeojsonPointItem
 
 
 class AverittSpider(scrapy.Spider):
     name = "averitt"
     item_attributes = {"brand": "Averitt Express", "brand_wikidata": "Q4828320"}
-    allowed_domains = ["averittexpress.com"]
+    allowed_domains = ["averitt.com"]
     start_urls = [
-        "https://www.averittexpress.com/resources/facility-locator",
+        "https://www.averitt.com/locations",
     ]
 
     def parse(self, response):
         states = response.xpath('//select[@name="state"]/option/@value').extract()
         for state in states:
-            if state == "0":
+            if state == "State":
                 continue
             yield scrapy.FormRequest(
-                url=response.urljoin("/resources/facility-locator/results"),
-                method="POST",
-                formdata={"state": state},
-                meta={"state": state},
+                url=response.urljoin(state.replace("..", "")),
                 callback=self.parse_state,
             )
 
     def parse_state(self, response):
         centers = response.xpath(
-            '//section[contains(@class, "content-main")]//div[@class="grid"]//a/@href'
+            '//*[@id="hs_cos_wrapper_widget_1613430615107_"]/b/a/@href'
         ).extract()
         for center in centers:
             yield scrapy.Request(response.urljoin(center), callback=self.parse_center)
```

Comment:
This page detection could probably be simplified by a SitemapSpider.
https://www.averitt.com/sitemap.xml

Reply:
You are right, I fixed it, thanks.